src/components/viewpointPage/Outliner.jsx

```diff
@@ -40,7 +40,14 @@ class Outliner extends React.Component {
         <div className="container-fluid">
           <div className="App-content row">
             {this.state.title
-              ? <Contributors viewpoint_id = {this.props.match.params.id} />
+              ? <div className="col-md-4 p-4 col-sm-12">
+                <Contributors viewpoint_id = {this.props.match.params.id} />
+                <button onClick={
+                  () => {
+                    this.applyDelete();
+                    window.confirm('Êtes-vous sûr de vouloir supprimer le point de vue ?');
+                  }}>Supprimer</button>
```

Comment:
Please use Lingui.js way to have the button title and the confirm message both in French and English.

Reply:
Asking for confirmation is a very good idea. But maybe you could explain more the consequences of saying yes.

Here are several classical advices for system messages wording: https://books.google.fr/books?id=PbD5lXI_d9AC&pg=PA6